server/validate/email/validate.go

```diff
@@ -49,6 +51,8 @@ type validator struct {
 	SMTPAddr string `json:"smtp_server"`
 	// Port of the SMTP server.
 	SMTPPort string `json:"smtp_port"`
+	// ServerName used in SMTP HELO/EHLO command.
+	ServerName string `json:"servername"`
```

Comment:
Please use something more descriptive instead of `servername`/`ServerName`, like `smtp_helo_host` and `SMTPHeloHost`.

Reply:
Renamed `servername/ServerName` to `smtp_helo_host/SMTPHeloHost`.